beetsplug/spotify.py

```diff
@@ -621,6 +624,9 @@ def _fetch_info(self, items, write, force):
             item['spotify_track_popularity'] = popularity
             audio_features = \
                 self.track_audio_features(spotify_track_id)
+            if audio_features is None:
+                self._log.debug('No audio features found for : {}', item)
```

Comment:
It seems like this one could be an `info`-level log—the user probably wants to know if the data is actually missing for a given song.

Reply:
Done